apiserver/plane/app/views/workspace/quick_link.py

```
@@ -0,0 +1,63 @@
+
+        serializer = WorkspaceUserLinkSerializer(quick_link)
+        return Response(serializer.data, status=status.HTTP_200_OK)
+    
+    @allow_permission([ROLE.ADMIN, ROLE.MEMBER, ROLE.GUEST], level="WORKSPACE")
+    def destroy(self, request, slug, pk):
+        quick_link = WorkspaceUserLink.objects.filter(pk=pk)
+
+        if quick_link: 
+            quick_link.delete()
+            return Response(status=status.HTTP_204_NO_CONTENT)
+        return Response({"detail": "Quick link not found."}, status=status.HTTP_404_NOT_FOUND)
```

Ruff (0.8.2)

Line too long (94 > 88)

(E501)